optimize/self-managed/optimize-deployment/advanced-features/import-guide.md

```diff
@@ -141,7 +141,7 @@ During execution, the following steps are performed:
 
 The import process is automatically scheduled in rounds by the `Import Scheduler` after startup of Optimize. In each import round, multiple `Import Services` are scheduled to run, each fetches data of one specific entity type. For example, one service is responsible for importing the historic activity instances and another one for the process definitions.
 
-For each service, it is checked if new data is available. Once all entities for one import service have been imported, the service starts to back off. To be more precise, before it can be scheduled again it stays idle for a certain period of time, controlled by the "backoff" interval and a "backoff" counter. After the idle time has passed, the service can perform another try to import new data. Each round in which no new data could be imported, the counter is incremented. Thus, the backoff counter will act as a multiplier for the backoff time and increase the idle time between two import rounds. This mechanism is configurable using the following properties:
+For each service, it is checked if new data is available. Once all entities for one import service have been imported, the service starts to back off. To be more precise, before it can be scheduled again it stays unassigned for a certain period of time, controlled by the "backoff" interval and a "backoff" counter. After the unassigned time has passed, the service can perform another try to import new data. Each round in which no new data could be imported, the counter is incremented. Thus, the backoff counter will act as a multiplier for the backoff time and increase the unassigned time between two import rounds. This mechanism is configurable using the following properties:
```

Comment:
This change is incorrect, it describes the import backoff mechanism

Reply:
ups, I will revert this then